transifex/utils.py

```diff
@@ -271,53 +271,57 @@ def merge_pr(self):
 
         retries = 0
         while retries <= self.MAX_MERGE_RETRIES:
-            try:
-                self.pr.merge(merge_method=self.merge_method)
-                logger.info('Merged [%s/#%d].', self.name, self.pr.number)
-                return True
-            except github.GithubException as e:
-                # Assumes only one commit is present on the PR.
-                statuses = self.pr.get_commits()[0].get_statuses()
-
-                # Check for any failing Travis builds. If any are found, notify the team
-                # and move on.
-                if any('travis' in s.context and s.state == 'failure' for s in statuses):
+            # Assumes only one commit is present on the PR.
+            statuses = self.pr.get_commits()[0].get_statuses()
```

Comment:
Will this be different for edx-platform since it uses jenkins instead of travis?

Reply:
actually i removed that too because even all tests pass this api call returns some pending which stop bot from merging the pr. (https://api.github.com/repos/edx/edx-platform/statuses/24ec64615c8ce7751d89cf0f828554ff0843afb8)
Started using the get combined status method which returns one status and makes the following api call (https://api.github.com/repos/edx/edx-platform/statuses/24ec64615c8ce7751d89cf0f828554ff0843afb8)